src/main/java/seedu/address/storage/JsonSerializableAddressBook.java

```diff
@@ -43,6 +46,9 @@ public JsonSerializableAddressBook(@JsonProperty("persons") List<JsonAdaptedPers
     public JsonSerializableAddressBook(ReadOnlyAddressBook source) {
         persons.addAll(source.getPersonList().stream().map(JsonAdaptedPerson::new).collect(Collectors.toList()));
         events.addAll(source.getEventList().stream().map(JsonAdaptedEvent::new).collect(Collectors.toList()));
+        linkedPersonsEntries.addAll(source.getPersonEventManager().getLinkedPersonsEntryList().stream()
+                .map(entry -> new JsonAdaptedLinkedPersonsEntry(entry))
```

Comment:
consider using `JsonAdaptedLinkedPersonsEntry::new` to follow style of previous 2 lines